docs/plugin_developers_guide/02_writing_plugins.rst

```diff
@@ -300,8 +300,8 @@ In C/C++ the *plugin_reconfigure* method is very similar, note however that the
    */
   void plugin_reconfigure(PLUGIN_HANDLE *handle, string& newConfig)
```

Comment:
Can we use a simple PLUGIN_HANDLE instead of PLUGIN_HANDLE* here too? (and remove the * from *handle to go with it?) Or is it mandatory to keep it like that due to the actual API behavior?

Reply:
No, we can not use a simple 'PLUGIN_HANDLE' for 'plugin_reconfigure()',
because the function pointers declared in
 * C/services/south/include/south_plugin.h:58 (for the south plugins)
 * C/services/north/include/north_plugin.h:53 (for the north plugins)

require a 'PLUGIN_HANDLE*'

(but for the filter plugins, it seems to be different ( C/common/include/filter_plugin.h:52)

This PR focuses on the North and the South plugins,

but we have to check this mix (between 'PLUGIN_HANDLE' and 'PLUGIN_HANDLE*') for the other types of plugins (as I mentioned here: https://github.com/fledge-iot/fledge/pull/1195#issuecomment-1761189195)